src/main/frontend/handler/editor.cljs

```diff
@@ -2758,7 +2758,7 @@
         (do (util/stop e)
             (autopair input-id key format nil))
 
-        hashtag?
+        (and hashtag? (re-find #"\s+$" value))
```

Comment:
I believe this solution works for non-latin languages. If I'm not correct, please provide a counterexample I can work against